README.md

```diff
@@ -142,7 +142,8 @@ Open HTTP API.
 
 Routes:
 
-- `/` 404 (Placeholder route)
+- `/` 404
```

Comment:
Should we remove this route? Should `/` return a simple HTML page listing the endpoints available?